clients/iOS/Grapevine/Grapevine/FeedManagementViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  AddFeedViewController.swift
+//  Grapevine
+//
+//  Created by Matthew Flickner on 11/24/15.
+//  Copyright © 2015 Grapevine. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import AlamofireObjectMapper
+import SwiftyJSON
+
+class FeedManagementViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var myFeeds: [String]!
+    var userToken: Token!
+    
+    @IBOutlet weak var tableView: UITableView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    
+    // Mark: - Table View Delegate Functions
+    
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        // Return the number of sections.
+        return 2
+    }
+    
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // Return the number of rows in the section.
+        if section == 0 {
+            return 1
+        }
+        else {
+            if myFeeds != nil {
+                return myFeeds.count
+            }
```

Comment:
Well you are saying if myFeeds is not nil then return myFeeds.count else 0. So why not something like

    myFeeds?.count ?? 0

Does that make sense? Looks more Swift-like to me.